src/OSSupport/Errors.cpp

```diff
@@ -22,7 +22,9 @@ AString GetOSErrorString( int a_ErrNo)
 
 	// According to https://linux.die.net/man/3/strerror_r there are two versions of strerror_r():
 
-	#if defined(__GLIBC__) && defined( _GNU_SOURCE) && !defined(ANDROID)  // GNU version of strerror_r()
+	// GNU version of strerror_r()
+	#if defined(__GLIBC__) && defined( _GNU_SOURCE) && !defined(ANDROID) \
+		|| defined(__CYGWIN__)
```

Comment:
These should be properly parenthesised to indicate the correct evaluation order.